book/website/community-handbook/contributing-guideline.md

```diff
@@ -0,0 +1,8 @@
+(contributing-guideline)=
+
+```{note}
+This page displays the [`CONTRIBUTING.md`](https://github.com/the-turing-way/the-turing-way/blob/main/CONTRIBUTING.md), a base file with guidelines for contributing to _The Turing Way_.
+```
+
+```{include} ../../../CONTRIBUTING.md
```

Comment:
This caused #3976 

I suppose we could hardwire the absolute URL for CoC

Reply:
I was wondering how many link on this page might be broken when navigating from the book. I can edit the contributing file to add full links. Sorry about this.

Reply:
It is all fixed now, I only commented at the end of the detective mission to see if something is broken in our CI or not.

 But it all worked as expected, it was just an oversight that the PR got merged with build issues.